recipes/dbus/1.x.x/conanfile.py

```diff
@@ -103,53 +75,51 @@ def _configure_cmake(self):
 
             self._cmake.definitions["DBUS_BUILD_X11"] = self.options.get_safe("with_x11", False)
             self._cmake.definitions["DBUS_WITH_GLIB"] = self.options.with_glib
-            self._cmake.definitions["DBUS_DISABLE_ASSERT"] = False
+            self._cmake.definitions["DBUS_DISABLE_ASSERT"] = is_apple_os(self.settings.os)
             self._cmake.definitions["DBUS_DISABLE_CHECKS"] = False
 
+            # Conan does not provide an EXPAT_LIBRARIES CMake variable for the Expat library.
+            # Define EXPAT_LIBRARIES to be the expat::expat target provided by Conan to fix linking.
+            self._cmake.definitions["EXPAT_LIBRARIES"] = "expat::expat"
+
             path_to_cmake_lists = os.path.join(self._source_subfolder, "cmake")
 
             self._cmake.configure(source_folder=path_to_cmake_lists,
                                   build_folder=self._build_subfolder)
         return self._cmake
 
     def build(self):
-        tools.replace_in_file(os.path.join(self._source_subfolder, "cmake", "CMakeLists.txt"),
-                              "project(dbus)",
-                              "project(dbus)\ninclude(../../conanbuildinfo.cmake)\nconan_basic_setup()")
-        if self.settings.os == "Windows":
-            cmake = self._configure_cmake()
-            cmake.build()
-        else:
-            autotools = self._configure_autotools()
-            autotools.make()
+        replace_in_file(
+            self,
+            os.path.join(self._source_subfolder, "cmake", "CMakeLists.txt"),
+            "project(dbus)",
+            "project(dbus)\ninclude(../../conanbuildinfo.cmake)\nconan_basic_setup()")
```

Comment:
Is there a particular reason for not using our usual `cmake-wrapper` ?

Reply:
Usual `cmake-wrapper`? You mean a `CMakeLists.txt` that just includes the source one with `add_subdirectory` while doing the Conan setup? That does seem like a better idea.

Reply:
I tried. The `CMakeLists.txt` files in this version of the project are poorly written and would require patching multiple things that incorrectly rely on the path of the top-level `CMakeLists.txt` file and not the current or project-level `CMakeLists.txt` location.

Reply:
There were only a couple of places I had to patch. I've included the patches and the `cmake-wrapper`.